src/config/tokens.config.ts

```diff
@@ -141,9 +143,11 @@ const RAW_TOKENS: ContractRegistry = {
   SMM_WEVE_USDC: '0xD9A4108CBB40a12dE16dFFDc54aE5065878816d7',
   SMM_OXD_USDC: '0xEaFB5Ae6eEa34954eE5e5a27B068B8705CE926a6',
   SMM_WFTM_CRV: '0xed7fd242ce91a541abcae52f3d617daca7fe6e34',
-  SMM_USDC_MIM: '0xbcab7d083cf6a01e0dda9ed7f8a02b47d125e682',
```

Comment:
This address was correct, the new one is repeated below and it points to wFTM/SCREAM

Reply:
fixed